cypress/integration/plugins/reports-dashboards/04-download.spec.js

```diff
@@ -86,7 +86,7 @@ describe('Cypress', () => {
     cy.get('#reportGenerationProgressModal');
   });
 
-  it('Download csv from saved search in-context menu', () => {
```

Comment:
is it flaky with security only?

could potentially do something similar here:  https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/884